[Design] 고객용 어플 입장 화면 레이아웃 추가

taek0622 opened the pull request:
## Motivation 🥳 (코드를 추가/변경하게 된 이유)
- 고객용 어플에서 첫 화면을 보여주기 위함

## Key Changes 🔥 (주요 구현/변경 사항)
- SplashView 추가

## ToDo 📆 (남은 작업)
- [ ] 버튼을 눌렀을 때, 초대 코드 입력 뷰로 넘어가는 기능 추가
- [ ] 화면 상단에 로고 추가
- [ ] 화면 배경색 변경

## ScreenShot 📷 (참고 사진)
<img src="https://user-images.githubusercontent.com/81027256/212759129-55bd209c-02e8-4ede-af3b-1c6b91df3556.png" width="300">

## To Reviewers 🙏 (리뷰어에게 전달하고 싶은 말)
- 간단한 레이아웃 추가 PR 입니다.
- 아주 간단한 코드이므로, 가볍게 확인해주셔도 좋습니다.

## Reference 🔗
- 없음

## Close Issues 🔒 (닫을 Issue)
- Close #9.

taek0622 committed:
Design:iphone:: #9 Add SplashView Layout

- 고객용 어플 입장 화면 추가

taek0622 committed:
Style:art:: #9 Add blank line at import part

- 내장 라이브러리와 써드파티 라이브러리를 구분하기 위한 공백 추가

creohwan commented on GiwazipClient/Views/SplashViewController.swift:
```diff
@@ -0,0 +1,38 @@
+        super.layout()
+
+        view.addSubview(enterButton)
+        enterButton.snp.makeConstraints { make in
+            make.left.equalTo(view.safeAreaLayoutGuide).offset(16)
+            make.right.bottom.equalTo(view.safeAreaLayoutGuide).offset(-16)
+            make.height.equalTo(50)
+        }
```
p3;
1. 혹시 make 대신 $0 이런 방법으로 layout 잡는 건 어떠할까요??! 저는 이게 더 편하긴 합니다. 내일 함께 이야기해보아요!
2. offset 대신 inset을 이용하면 한 번에 처리할 수 있습니다!

```suggestion
        enterButton.snp.makeConstraints {
            $0.left.bottom.right.equalTo(view.safeAreaLayoutGuide).inset(16)
            $0.height.equalTo(50)
        }
```

ikanic replied:
> p3;
> 
> 1. 혹시 make 대신 $0 이런 방법으로 layout 잡는 건 어떠할까요??! 저는 이게 더 편하긴 합니다. 내일 함께 이야기해보아요!
> 2. offset 대신 inset을 이용하면 한 번에 처리할 수 있습니다!

오 저도 이런 식으로 할 수는 없나 고민만 하고 방법을 못찾고 있었는데, 디너가 좋은 방법을 제시해 주셨네요.
훨씬 깔끔하고 좋은 것 같네요. 바로 반영하겠습니다. 굳~

taek0622 committed:
Style:art:: #9 Modify SnapKit Code Style

- 클로저 안에서 make라는 이름으로 사용하던 부분을 $0로 변경
- 비슷한 역할을 하는 코드들을 offset이 아닌 inset을 이용하여 묶음

creohwan commented on GiwazipClient/Views/SplashViewController.swift:
```diff
@@ -0,0 +1,37 @@
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .semibold)
+        $0.backgroundColor = .gray
+        $0.layer.cornerRadius = 16
+
```
p1; 
공백 지워주시면 감사하겠습니다! 고생하셨습니다 : )

ikanic replied:
> p1; 공백 지워주시면 감사하겠습니다! 고생하셨습니다 : )

해당 공백은 선언부와 반환부를 분리하기 위해 일부러 남긴 공백입니다~

creohwan replied:
> > p1; 공백 지워주시면 감사하겠습니다! 고생하셨습니다 : )
> 
> 해당 공백은 선언부와 반환부를 분리하기 위해 일부러 남긴 공백입니다~

오잉! 그런 게 있었군요! 혹시 새로 생긴 컨벤션일까요...?!

ikanic replied:
> > > p1; 공백 지워주시면 감사하겠습니다! 고생하셨습니다 : )
> > 
> > 
> > 해당 공백은 선언부와 반환부를 분리하기 위해 일부러 남긴 공백입니다~
> 
> 오잉! 그런 게 있었군요! 혹시 새로 생긴 컨벤션일까요...?!

아뇨 이 부분에 대해서는 정해진 컨벤션이 따로 있진 않아요.

ikanic replied:
> p1; 공백 지워주시면 감사하겠습니다! 고생하셨습니다 : )

개인의 코드 스타일이기도 하고, 따로 컨벤션이 없긴 하지만 우선은 프로젝트 통일성을 위해서 해당 부분 공백 삭제했습니다~

taek0622 committed:
Style:art:: Delete empty line
